install.php

```diff
@@ -207,6 +250,186 @@ public function installOrUpdate($parent)
 	}
 
 	/**
+	 * Method to process SQL updates previous to the install process
+	 *
+	 * @param   object  $parent  Class calling this method
+	 *
+	 * @return  boolean          True on success
+	 */
+	public function preprocessUpdates($parent)
```

Comment:
shouldn't this cause problems, like when you are creating new column... and then run it again, it will crash on second run since you already have created it